CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
-# Contributing to Docker
+# Contributing to the Moby Project
```

Comment:
Maybe this is a good opportunity to stop using the gerund form for titles "Contribute to the Moby Project" perhaps?